DRODLib/CharacterCommand.h

```diff
@@ -184,6 +184,17 @@ namespace ScriptFlag
 		Stalwart=4
 	};
 
+	enum GotoSmartType
```

Comment:
@EvidentlyCube , I realize I'm not sure whether you were being facetious about actually adding all of these potential ways to jump around in the code.  Do you think it's a good idea to actually have all of these?

Reply:
@Hypexion , my kneejerk reaction here would be to add these as special/predefined options to the "Goto <label>" command.  That is, rather than having a new command, add these as options in the Goto listbox (probably with negative values instead of positive).

Reason being it reads much easier if the command says "Goto LastIf" instead of "Goto Smart LastIf".  Also, we don't need another command, really.

I'm not trying to be prescriptive here; just wanting to start a conversation to see what you've already considered as you designed and implemented this functionality.

Reply:
I think there's going to be a tradeoff between keeping the number of commands as low as possible, and the engineering issues that might come up with using the existing Goto structure. The command execution side probably isn't too bad, but I think the UI side will run into issues. Currently, the widget for selecting Goto labels is also used when selecting labels for question options, and I'm not sure if we want the functionallity offered through that path.

Reply:
> ...I think the UI side will run into issues. Currently, the widget for selecting Goto labels is also used when selecting labels for question options, and I'm not sure if we want the functionality offered through that path.

Ah, I'd forgotten about that.  I agree it probably wouldn't make sense to populate question options with the smart goto options.